packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -38,29 +38,30 @@ Below is an example of how these elements would be used within a `nimble-table`:
 
 ```HTML
 <nimble-table>
-    <nimble-table-column-icon field-name="status" key-type="string">
+    <nimble-table-column-icon field-name="status" key-type="string" label-hidden>
         Status
         <nimble-mapping-icon key="fail" icon="nimble-icon-xmark" severity="error" label="Failed"></nimble-mapping-icon>
         <nimble-mapping-icon key="error" icon="nimble-icon-xmark" severity="error" label="Errored"></nimble-mapping-icon>
         <nimble-mapping-icon key="pass" icon="nimble-icon-check" severity="success" label="Passed"></nimble-mapping-icon>
         <nimble-mapping-spinner key="running" label="Running"></nimble-mapping-spinner>
     </nimble-table-column-icon>
-    <nimble-table-column-enum-text field-name="errorCode" key-type="number">
+    <nimble-table-column-icon field-name="errorCode" key-type="number">
         Error Code
         <nimble-mapping-text key="1" label="A bad thing happened"></nimble-mapping-text>
         <nimble-mapping-text key="2" label="A worse thing happened"></nimble-mapping-text>
         <nimble-mapping-text key="3" label="A terrible thing happened"></nimble-mapping-text>
-    </nimble-table-column-enum-text>
+    </nimble-table-column-icon>
     <nimble-table-column-icon field-name="archived" key-type="boolean">
         Archived
         <nimble-mapping-icon key="true" icon="nimble-icon-database" label="Archived"></nimble-mapping-icon>
+        <nimble-mapping-icon key="false" label="Not archived"></nimble-mapping-icon>
```

Comment:
Note: I added this `nimble-mapping-icon` element here since that makes it follow the best practices of having a mapping for each possible record value.